src/sample/complete/TicketService.java

```diff
@@ -1,40 +1,66 @@
 package sample.complete;
 
-import java.time.Instant;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Random;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
 public class TicketService {
 
-	private static Map<Integer, Ticket> tickets;
+    private static Map<Integer, Ticket> tickets;
 
-	public static void main(String[] args) {
+    public static void main(String[] args) {
 
-		tickets = new HashMap<>();
+        tickets = new HashMap<>();
 
-		// generate 10 tickets
-		IntStream.rangeClosed(1, 10)
-				.forEach(ticket -> {
-					long unixTimestamp = System.currentTimeMillis() / 1000;
-					tickets.put(ticket, new Ticket("Event-" + ticket, ticket, unixTimestamp));
+        // generate 10 tickets
+        IntStream.rangeClosed(1, 10)
+                .forEach(ticket -> {
+                    long unixTimestamp = System.currentTimeMillis() / 1000;
+                    tickets.put(ticket, new Ticket("Event-" + ticket, ticket, unixTimestamp));
 
-					try {
-						// Sleep for 500 ms
-						Thread.sleep(500);
-					} catch (InterruptedException e) {
-						Thread.currentThread().interrupt();
-					}
-				});
+                    try {
+                        // Sleep for 500 ms
+                        Thread.sleep(500);
+                    } catch (InterruptedException e) {
+                        Thread.currentThread().interrupt();
+                    }
+                });
 
-		var ticket = getTicketById(1);
-		System.out.println(ticket);
-
-	}
-
-	private static Ticket getTicketById(int id) {
-		return tickets.get(id);
-	}
+        var ticket = getTicketById(1);
+        System.out.println(ticket);
+//testing getTicketsByStadiumSector function
+        Map<Integer, Ticket> ticketByStadiumSector = getTicketsByStadiumSector('A');
```

Comment:
the var called **ticketByStadiumSector**, the method getTicket**_s_**ByStadiumSector
jsut use the same style please))